app/views/school/home/_closed_banner_single.html.erb

```diff
@@ -0,0 +1,10 @@
+<p class="govuk-body">
+  From <span class="app-no-wrap">September 2020</span> to <span class="app-no-wrap">July 2021</span>, your school received
```

Comment:
instead of school — can you use <%= @school.institution_type %>

Reply:
I'll add it